Makefile

```diff
@@ -220,7 +220,18 @@ run_tests.lighthouse_performance: ## Runs the lighthouse performance tests for t
 	@echo '-----------------------------------------------------------------------'
 	$(MAKE) stop
 
+run_tests.check_e2e_tests_are_captured_in_ci: ## Runs the check to ensure that all e2e tests are captured in CI
+	docker compose up dev-server -d --no-deps
+	$(SHELL_PREFIX) dev-server python -m scripts.check_e2e_tests_are_captured_in_ci
+	$(MAKE) stop
+
+run_tests.typescript_tests: ## Runs the typescript tests
+	docker compose up dev-server -d --no-deps
+	$(SHELL_PREFIX) dev-server python -m scripts.typescript_checks $(PYTHON_ARGS)
+	$(MAKE) stop
+
 OS_NAME := $(shell uname)
+sharding_instances := 3
```

Comment:
Why do we need this here?

Also, please define the variables on top*

Reply:
@Shivkant-Chauhan Is this resolved?

Reply:
this is a flag for the e2e tests. refer here from the python workflow -- https://github.com/oppia/oppia/blob/develop/scripts/run_e2e_tests.py#L59.

moved it to the top. thanks!